WordPress/Docs/WhiteSpace/CastStructureSpacingStandard.xml

```diff
@@ -0,0 +1,19 @@
+<documentation title="Cast Structure Spacing">
+    <standard>
+    <![CDATA[
+    Surround type casts with spaces.
```

Comment:
This sniff only checks the whitespace _before_ a type cast. There is an upstream sniff which checks the whitespace after and within.

```suggestion
    A type cast should be preceded by whitespace.
```